css/style.css

```diff
@@ -0,0 +1,95 @@
+body {
+  /* background: #a5a5a5;
+    color: #fff; */
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.container {
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+h1 {
+  text-align: center;
+}
+
+#description {
+  text-align: center;
+  font-size: 20px;
+}
```

Comment:
kindly use the class selector because it can not be affected by the other selector or be overwritten